src/controller/python/chip/interaction_model/Delegate.h

```diff
@@ -59,15 +59,15 @@ struct __attribute__((packed)) AttributeWriteStatus
     chip::NodeId nodeId;
     uint64_t appIdentifier;
     uint32_t protocolId;
-    uint16_t protocolCode;
+    uint32_t protocolCode;
     chip::EndpointId endpointId;
     chip::ClusterId clusterId;
     chip::FieldId fieldId;
 };
 static_assert(std::is_same<chip::EndpointId, uint16_t>::value && std::is_same<chip::ClusterId, uint32_t>::value &&
                   std::is_same<chip::FieldId, uint32_t>::value,
               "Members in AttributeWriteStatus does not match interaction_model/delegate.py");
-static_assert(sizeof(AttributeWriteStatus) == 8 + 8 + 4 + 2 + 2 + 4 + 4, "Size of AttributeWriteStatus might contain padding");
+static_assert(sizeof(AttributeWriteStatus) == 8 + 8 + 4 + 4 + 2 + 4 + 4, "Size of AttributeWriteStatus might contain padding");
```

Comment:
You should change `src/controller/python/chip/interaction_model/delegate.py` for this line as well.

Reply:
Done, thanks